src/test/java/net/adoptium/GreetingResourceTest.java

```diff
@@ -10,12 +10,22 @@
 public class GreetingResourceTest {
 
     @Test
-    public void testHelloEndpoint() {
-        given()
+    public void testHelloEndpointEnglish() {
+        given().header("Accept-Language", "en-US") //chrome and edge sends with a "-"
           .when().get("/hello")
           .then()
              .statusCode(200)
-             .body(is("Hello World"));
+             .body(is("Hello World!"));
     }
 
+    @Test
+    public void testHelloEndpointGerman() {
+        given().header("Accept-Language", "de,en-US;q=0.7,en;q=0.3") //chrome and edge sends with a "-"
+                .when().get("/hello")
+                .then()
+                .statusCode(200)
+                .body(is("Hallo Welt!"));
```

Comment:
Rest Assured is an odd choice for testing a website. It's primarily geared towards testing REST endpoints (hence the name). For a website, you need something else that also involves a Browser. Selenium is the veteran here, Playwright is relatively new and looks great.